api/db.py

```diff
@@ -56,6 +56,29 @@ def fetchSingleEdition(self, editionID, showAll=False):
             )\
             .filter(Edition.id == editionID).first()
 
+    def fetchSortedEditions(self, sortField: str, sortDirection: str, limit: int):
```

Comment:
👍 to the code, feels like the name is a bit more general than the function though. How would you feel about one of these options?
1. Update the function name to reflect how they're sorted.
2. Make this a bit more generic so that `sortField` could be any valid `Edition` field. I'm not 100% sure that would make sense (I like the sanity check here).

Reply:
I just changed the function name, but maybe not in a way that really solves option 1. For option 2 (making this more generic), I'm not sure I want to implement the magic to allow arbitrary sort fields passed in, but I did add to the docstring a note to consider it, thoughts?